app/forms/flood_risk_engine/steps/base_address_form.rb

```diff
@@ -96,7 +104,7 @@ def find_by_postcode
       end
 
       def initialize(model, enrollment)
-        @address_to_assign = Address.new
+        @assignable_address = Address.new
```

Comment:
Is a check needed here in case an address already exists (for example, on a backward journey)?